include/node.h

```diff
@@ -306,6 +319,10 @@ class Node : public NodeBase<Tdim> {
   //! A general velocity (non-Cartesian/inclined) constraint is specified at the
   //! node
   bool generic_boundary_constraints_{false};
+  //! Absorbing constraints
```

Comment:
Could we pass this instead of storing it in the node class? As this would increase the size of each node class by a 256 bytes unnecessarily.

Reply:
`absorbing_constraint_` previously removed